### DISCUSSION TITLE
Is there anyway to get the cropped Image as output?

### DISCUSSION THREAD

**Discussion by fzsf163** [pikasojs/pikaso#107] [Q&A] [answered]
### Is there any way to get the cropped Image just like getting the Rect ?

I was hoping if there was anyway to get the cropped image. The cropper works fine. It returns the position of the cropper. But is it possible to get the image where the cropper is situated ?

**Comment by raminious** [comment, marked as answer]
Yes, you need to call `export.toImage` 

```ts
const editor = new Pikaso()

// do your crop

const image = editor.export.toImage()
```

https://pikaso.app/api/classes/Export.html#toImage

**Reply by fzsf163** [reply]
amazing. Simply amazing.
